lib/commonmarker/config.rb

```diff
@@ -7,23 +7,28 @@ module Config
     OPTS = {
       parse: {
         DEFAULT: 0,
+        SOURCEPOS: (1 << 1),
+        UNSAFE: (1 << 17)
```

Comment:
To keep it inline with the deprecated SAFE flag, `cmark-gfm.h` defines UNSAFE after `1 << 3`, so I've sorted it here accordingly.

Reply:
Looks like a comma is missing.

Reply:
🤦